easy-ui-react/src/components/Placeholder/Placeholder.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Placeholder } from "./Placeholder";
+
+describe("Placeholder", async () => {
+  it("should render the placeholder message", () => {
+    render(<Placeholder status="success" message={<p>Test message</p>} />);
+    expect(screen.getByText("Test message")).toBeInTheDocument();
+  });
+});
```

Comment:
i know this is just a placeholder, but when it comes to tests do you have an idea in mind for how they should be written for the components that will live in easy-ui?

Reply:
good question. i'm not sure exactly yet on the specifics. probably another good thing to figure out conventions for as we go